src/ert/_c_wrappers/enkf/site_config.py

```diff
@@ -63,106 +123,53 @@ def from_config_dict(self, config_dict):
                     config_file=job[ConfigKeys.PATH],
                     private=False,
                     name=job[ConfigKeys.NAME],
-                    license_root_path=__license_root_path,
+                    license_root_path=license_root_path,
                 )
                 new_job.convertToCReference(None)
                 ext_job_list.add_job(job[ConfigKeys.NAME], new_job)
-            except (ValueError, OSError):
-                print(f"WARNING: Unable to create job from {job[ConfigKeys.PATH]}")
+            except (ValueError, OSError) as e:
+                print(f"WARNING: Unable to create job from {job[ConfigKeys.PATH]}: {e}")
 
         for job_path in config_dict.get(ConfigKeys.INSTALL_JOB_DIRECTORY, []):
             if not os.path.isdir(job_path):
                 print(f"WARNING: Unable to locate job directory {job_path}")
                 continue
             files = os.listdir(job_path)
+            print(files)
             for file_name in files:
                 full_path = os.path.join(job_path, file_name)
                 if os.path.isfile(full_path):
                     try:
                         new_job = ExtJob(
                             config_file=full_path,
                             private=False,
-                            license_root_path=__license_root_path,
+                            license_root_path=license_root_path,
                         )
                         new_job.convertToCReference(None)
                         ext_job_list.add_job(new_job.name(), new_job)
                     except (ValueError, OSError):
-                        print(f"WARNING: Unable to create job from {full_path}")
+                        print(
+                            f"WARNING: Unable to create job from {full_path}"
+                            f" in directory {job_path}"
+                        )
 
         ext_job_list.convertToCReference(None)
```

Comment:
as you can imagine, i don't really know what `converToCReference` does, or why we need it. do i assume correctly that we still need it here? you removed C code for site config, but the `ExtJobList` class is still a class based on C code?

Reply:
It should not be here now that it is no longer part of the site_config struct. If I understood it correctly. Will also remove the other calls to it on ExtJob, because it is being called automatically when adding it to the list.

Reply:
okay cool : )